i18n-populator.config.json

```
@@ -1,6 +1,17 @@
 {
   "basePath": "example",
-  "translationEngines": ["google", "bing", "libreTranslate"],
+  "translationEngines": [
+    {
+      "name": "google"
+    },
+    {
+      "name": "bing"
+    },
+    {
+      "name": "libreTranslate",
+      "mirrors": ["mirror1", "mirror2", "mirror3"]
```

By the moment it can be empty, the "mirrorNumber" thing was just an example
```suggestion
      "mirrors": []
```